cc/mcts_player_test.cc

```diff
@@ -290,6 +302,48 @@ TEST_F(MctsPlayerTest, ParallelTreeSearch) {
   EXPECT_EQ(0, CountPendingVirtualLosses(root));
 }
 
+TEST_F(MctsPlayerTest, DontPassOnEmptyLosingBoard) {
+  MctsPlayer::Options options;
+  auto player = CreateBasicPlayer(options);
+  auto* root = player->root();
+  auto board = TestablePosition(kOneStoneBoard, Color::kWhite);
```

Comment:
Can you add comment. // Search board with one black stone, white to play.

Reply:
Done.